packages/data/walkthroughs/9.9.9.json

```diff
@@ -406,15 +412,1149 @@
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "Yes",
+          "answerValue": "true"
+        },
+        {
+          "answerDisplayText": "No",
+          "answerValue": "false"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P1",
+          "answerValue": "true",
+          "nextNavigateTo": "P2"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "R30"
+        }
+      ]
+    },
+    "P2": {
+      "walkthroughItemType": "multiChoice",
+      "questionText": "How many <defined-term>storeys</defined-term> does this <defined-term>dwelling unit</defined-term> have?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "1 storey",
+          "answerValue": "1"
+        },
+        {
+          "answerDisplayText": "1 storey + <defined-term>basement</defined-term>",
+          "answerValue": "1B"
+        },
+        {
+          "answerDisplayText": "2 stories",
+          "answerValue": "2"
+        },
+        {
+          "answerDisplayText": "2 stories + <defined-term>basement</defined-term>",
+          "answerValue": "2B"
+        },
+        {
+          "answerDisplayText": "3 stories",
+          "answerValue": "3"
+        },
+        {
+          "answerDisplayText": "3 stories + <defined-term>basement</defined-term>",
+          "answerValue": "3B"
+        },
+        {
+          "answerDisplayText": "More than 3 stories",
+          "answerValue": "4"
+        }
+      ],
+      "nextNavigationLogic": [
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P2",
+          "answerValue": "1",
+          "nextNavigateTo": "P10"
+        },
+        {
+          "nextLogicType": "equals",
+          "answerToCheck": "P2",
+          "answerValue": "4",
+          "nextNavigateTo": "R30"
+        },
+        {
+          "nextLogicType": "fallback",
+          "nextNavigateTo": "P3"
+        }
+      ]
+    },
+    "P3": {
+      "walkthroughItemType": "multiChoiceMultiple",
+      "answersAreDynamic": true,
+      "questionText": "Which <defined-term>storeys</defined-term> within the <defined-term>dwelling unit</defined-term> are provided with an egress or <defined-term>exit</defined-term> doors?",
+      "possibleAnswers": [
+        {
+          "answerDisplayText": "<defined-term>Basement</defined-term>",
+          "answerValue": "B",
+          "showAnswerIf": [
+            {
+              "showAnswerLogicType": "equals",
+              "answerToCheck": "P2",
+              "answerValue": "1B"
+            },
+            {
+              "showAnswerLogicType": "equals",
+              "answerToCheck": "P2",
+              "answerValue": "2B"
+            },
+            {
+              "showAnswerLogicType": "equals",
+              "answerToCheck": "P2",
+              "answerValue": "3B"
```

Comment:
Can a building with more than 3 stories have a basement? Is that something we should be concerned about?